src/coreclr/hosts/bundle/file_entry.cpp

```diff
@@ -38,13 +37,11 @@ file_entry_t file_entry_t::read(reader_t &reader)
 
     if (!entry.is_valid())
     {
-        trace::error(_X("Failure processing application bundle; possible file corruption."));
```

Comment:
The `trace:error()` routines were no-ops, since the backing functions were not ported. Therefore, I removed the calls from the prototype, because the failure can be easily identified in the debugger (which we needed anyway, since there was no trace output).

Reply:
OK - for prototype this is fine.